pegasus/sites.v3/code.org/public/pd-workshop-survey/splat.haml

```diff
@@ -4,6 +4,8 @@ title: "Professional Development Workshop Survey"
 :ruby
   dont_cache
   enrollment_code = request.splat_path_info[1..-1]
+  redirect (request.scheme || 'http') + ':' + CDO.studio_url("/pd/workshop_survey/#{enrollment_code}")
```

Comment:
You can also use [CDO.default_scheme](https://github.com/code-dot-org/code-dot-org/blob/staging/deployment.rb#L243)

Like
```
CDO.studio_url("/pd/workshop_survey/#{enrollment_code}", CDO.default_scheme)
```
or even 
```
CDO.studio_url("/pd/workshop_survey/#{enrollment_code}", request.scheme || CDO.default_scheme)
```